lms/djangoapps/discussion/rest_api/views.py

```diff
@@ -1146,6 +1146,7 @@ class ReplaceUsernamesView(APIView):
 
     """
 
+    authentication_classes = (JwtAuthentication,)
```

Comment:
@salman2013 why was this added back in? It looks like there was a mismatch of response codes for a test. 403 vs 401.  I think the more correct thing to do is to update the test in this case.  We saw [this issue](https://github.com/openedx/edx-platform/pull/32802#issuecomment-1702817880) here as well.